tools/bls-signature-creator/main.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"encoding/hex"
+	"fmt"
+	"log/slog"
+	"os"
+	"strings"
+
+	"github.com/cloudflare/circl/sign/bls"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/crypto"
+
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	optionPrivateKey = &cli.StringFlag{
+		Name:  "private-key",
+		Usage: "BLS private key as hex encoded string (with optional 0x prefix). Must be a valid hex string representing a BLS private key.",
+	}
+
+	optionEthAddress = &cli.StringFlag{
```

Comment:
We can make this more generic by just saying payload here and then docs can say that users need to use their Eth Address